tests/test_instanceconnectionmanager_unit.py

```diff
@@ -46,3 +47,14 @@ def test_InstanceConnectionManager_wrong_connection_string():
     loop = asyncio.new_event_loop()
     with pytest.raises(CloudSQLConnectionError):
         InstanceConnectionManager("test-project:test-region", loop)
+
+
+def test_InstanceConnectionManager_perform_refresh():
+    """
+    Test to check whether _perform_refresh can run without any errors given a
+    live testing string.
+    """
+    test_connection_string = os.environ["CLOUD_SQL_CONNECTION_STRING"]
```

Comment:
Is this something that you can assume is always in the environment?

Reply:
No, it's probably not - it would be a good idea to check if this is unset and either throw an error or skip the test (not sure which would be best for Python).

Also, please try to use `"INSTANCE_CONNECTION_NAME"` since this is how the SQL docs refer to it.